python/felis/tap.py

```diff
@@ -238,6 +282,20 @@ def visit_constraints(self, schema_obj: Schema) -> tuple:
         return all_keys, all_key_columns
 
     def visit_table(self, table_obj: Table, schema_obj: Schema) -> tuple:
+        """Visit a table object and build its TAP_SCHEMA representation.
+
+        Parameters
+        ----------
+        table_obj : `felis.datamodel.Table`
```

Comment:
You are adding types here but not in other places?

Reply:
I missed a few. I had intended to remove all of them and will strip out any that are still remaining.